resources/views/auth/passwords/confirm.blade.php

```diff
@@ -0,0 +1,49 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Confirm Password') }}</div>
```

Comment:
Ese texto debe ir en lang
```suggestion
                <div class="card-header">@lang('confirm.confirm')</div>
```
(ejemplo)